contracts/Manager.sol

```diff
@@ -761,10 +991,14 @@ contract Manager is UUPSOwnableUpgradeable, UsingRegistryUpgradeable {
      * @param group The group to deprecate.
      */
     function _deprecateGroup(address group) private {
-        if (!activeGroups.remove(group)) {
-            revert GroupNotActive(group);
+        bool activeGroupsRemoval = activeGroups.remove(group);
+        bool specificGroupsRemoval = specificStrategyVotedGroups.remove(group);
+        if (!activeGroupsRemoval && !specificGroupsRemoval) {
+            revert GroupNotActiveNorSpecific(group);
         }
 
+        delete specificStrategyTotalStCeloVotes[group];
```

Comment:
Can we just 0 this? Without any transfer of votes or increase in another strategy's amount?